app/src/main/AndroidManifest.xml

```diff
@@ -2,14 +2,21 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.codecool.lolapp">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+
     <application
+        android:name=".di.MyCustomApp"
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/icon"
         android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
+        android:roundIcon="@drawable/icon"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true">
+        <activity android:name=".activities.MainActivity"
+            android:theme="@style/Theme.AppCompat.NoActionBar"></activity>
+        <activity android:name=".activities.SplashScreenActivity"
```

Comment:
It's okay to do it like this, especially when you need to download configurations before the app starts, but I wanted to mention that you can include a simple layout file into the manifest to make a splash, without the need to create a whole activity